internal/querynode/data_sync_service.go

```diff
@@ -272,20 +271,6 @@ func (dsService *dataSyncService) startPartitionFlowGraph(partitionID UniqueID,
 func (dsService *dataSyncService) removePartitionFlowGraph(partitionID UniqueID) {
```

Comment:
why release partition fg is still needed?

Reply:
this pr doesn't contains the modifies of flow graph